ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -33,6 +33,7 @@ StatusDialog {
     property string preSelectedHoldingID
     property int preSelectedHoldingType: Constants.TokenType.Unknown
     property int preSelectedSendType
+    property string ownerTokenCommunityId
```

Comment:
This addition in here (global and generic `SendModal` component) makes me wonder why do we need to treat this case as an special one if it's really just another collectible transaction, no?

Reply:
Good catch, it is a leftover. I will remove it.